ribs/visualize.py

```diff
@@ -137,7 +137,7 @@ def grid_archive_heatmap(archive,
         archive (GridArchive): A 2D :class:`~ribs.archives.GridArchive`.
         ax (matplotlib.axes.Axes): Axes on which to plot the heatmap.
             If ``None``, the current axis will be used.
-        transpose_bcs (bool): By default, the first BC in the archive will
+        transpose_measures (bool): By default, the first measure in the archive will
```

Comment:
Not sure if this makes sense. Maybe "the first measure function in the archive"?

Reply:
I think this is fine as is.

Reply:
I think this is fine as is